src/components/employees.svelte

```diff
@@ -9,139 +9,175 @@
-</script>
-
-<Employee
-  givenName="Thomas"
-  familyName="Brustad"
-  position="Rådgiver"
-  phone="+47 402 86 277"
-  email="thomas@bjerk.io"
-  image={thomas}
-  socialMedia={[
-    { type: 'linkedin', url: 'https://www.linkedin.com/in/thomasbrustad/' },
-    { type: 'github', url: 'https://github.com/HanBrustad' }
-  ]}
-/>
-
-<Employee
-  givenName="Simen A. W."
-  familyName="Olsen"
-  position="Daglig leder"
-  phone="+47 95 30 80 87"
-  email="simen@bjerk.io"
-  image={simen}
-  socialMedia={[
-    { type: 'linkedin', url: 'https://www.linkedin.com/in/simenandre/' },
-    { type: 'github', url: 'https://github.com/simenandre' }
-  ]}
-/>
 
-<Employee
-  givenName="Carl Oscar"
-  familyName="Petterson"
-  position="Rådgiver"
-  phone="+47 908 59 851"
-  email="cop@bjerk.io"
-  image={cop}
-  socialMedia={[
+  const employees = [
+    // Thomas
     {
-      type: 'linkedin',
-      url: 'https://www.linkedin.com/in/copetterson/'
+      givenName: 'Thomas',
+      familyName: 'Brustad',
+      position: 'Rådgiver',
+      phone: '+47 402 86 277',
+      email: 'thomas@bjerk.io',
+      image: thomas,
+      socialMedias: {
+        linkedin: {
+          url: 'https://www.linkedin.com/in/thomasbrustad/'
+        },
+        github: {
+          url: 'https://github.com/HanBrustad'
+        }
+      }
     },
+    // Simen
     {
-      type: 'github',
-      url: 'https://github.com/copetterson'
-    }
-  ]}
-/>
-
-<Employee
-  givenName="Brage Sekse"
-  familyName="Aarset"
-  position="Programvareutvikler"
-  phone="+47 464 65 823"
-  email="brage@bjerk.io"
-  image={brage}
-  socialMedia={[
-    { type: 'linkedin', url: 'https://www.linkedin.com/in/braaar/' },
-    { type: 'github', url: 'https://github.com/braaar' }
-  ]}
-/>
-
-<Employee
-  givenName="Markus"
-  familyName="Moripen"
-  position="Programvareutvikler"
-  phone="+47 970 30 084"
-  email="markus@bjerk.io"
-  image={markus}
-  socialMedia={[
-    { type: 'linkedin', url: 'https://www.linkedin.com/in/moripen/' },
-    { type: 'github', url: 'https://github.com/moripen' }
-  ]}
-/>
-
-<Employee
-  givenName="Simon"
-  familyName="Rugland"
-  position="Programvareutvikler"
-  phone="+47 474 46 841"
-  email="js@bjerk.io"
-  image={simon}
-  socialMedia={[
-    { type: 'linkedin', url: 'https://www.linkedin.com/in/simon-rugland/' },
-    { type: 'github', url: 'https://github.com/simonrugland' }
-  ]}
-/>
-
-<Employee
-  givenName="Sandra Viktoria"
-  familyName="Toftemo"
-  position="Programvareutvikler"
-  phone="+47 406 30 694"
-  email="sandra.toftemo@bjerk.io"
-  image={sandra}
-  socialMedia={[
+      givenName: 'Simen A. W.',
+      familyName: 'Olsen',
+      position: 'Daglig leder',
+      phone: '+47 95 30 80 87',
+      email: 'simen@bjerk.io',
+      image: simen,
+      socialMedias: {
+        linkedin: {
+          url: 'https://www.linkedin.com/in/simenandre/'
+        },
+        github: {
+          url: 'https://github.com/simenandre'
+        }
+      }
```

Comment:
```suggestion
      linkedin: 'https://www.linkedin.com/in/simenandre/',
      github: 'https://github.com/simenandre',
```

Reply:
I would either go for something simple like this or make this an array that can be passed directly to the employee component.